code/modules/clothing/spacesuits/rig/suits/light.dm

```diff
@@ -72,20 +72,31 @@
 	siemens_coefficient = 0
 
 /obj/item/weapon/rig/light/ninja
-	name = "ominous suit control module"
-	suit_type = "ominous"
-	desc = "A unique, vaccum-proof suit of nano-enhanced armor designed specifically for Spider Clan assassins."
+	name = "stealth suit control module"
+	suit_type = "stealth suit"
+	desc = "A unique, vaccum-proof suit of nano-enhanced armor designed specifically for stealth operations."
 	icon_state = "ninja_rig"
-	armor = list(melee = 50, bullet = 15, laser = 30, energy = 10, bomb = 25, bio = 100, rad = 30)
-	emp_protection = 40 //change this to 30 if too high.
+	armor = list(melee = 50, bullet = 35, laser = 35, energy = 30, bomb = 25, bio = 100, rad = 30)
+	emp_protection = 40
 	slowdown = 0
 
 	chest_type = /obj/item/clothing/suit/space/rig/light/ninja
 	glove_type = /obj/item/clothing/gloves/rig/light/ninja
 	helm_type =  /obj/item/clothing/head/helmet/space/light/ninja
+	boot_type = /obj/item/clothing/shoes/rig/light/ninja
 
 	req_access = list(access_syndicate)
+	initial_modules = list(
+		/obj/item/rig_module/teleporter,
+		/obj/item/rig_module/vision,
+		/obj/item/rig_module/voice,
+		/obj/item/rig_module/ai_container,
+		/obj/item/rig_module/datajack,
+		/obj/item/rig_module/self_destruct
+	)
+	..()
```

Comment:
The fact that this compiles is great! Anyways, this line shouldn't be here. Plz delet.